common/test/views/support/cleaner/AmpEmbedCleanerTest.scala

```diff
@@ -64,6 +80,34 @@ class AmpEmbedCleanerTest extends FlatSpec with Matchers {
     clean(document)
   }
 
+
+  private def cleanDocumentWithCommentEmbed(): Document = {
+
+    val doc = <html>
+      <body>
+      <figure class="element element-comment" data-canonical-url="https://discussion.theguardian.com/comment-permalink/88222201">
+        <div class="d2-comment-embedded" itemtype="http://schema.org/Comment">
+          <div class="d2-left-col">
+            <a href="https://profile.theguardian.com/user/id/12345678">
+              <img class="d2-avatar" src="https://avatar.guim.co.uk/user/15301515" height="40" width="40" alt="User avatar for fooBar"/>
+            </a>
+          </div>
+          <div class="d2-right-col">
+          </div>
+          <div class="d2-permalink">
+          </div>
+          <div class="d2-body" itemprop="text">
+            <p>foo bar foo foo bar</p>
+          </div>
+        </div>
+      </figure>
+      </body>
+      </html>.toString()
```

Comment:
I've reduced the size of this html document to more closely match other docs in the test file. Hopefully this eliminates the need of making it an external file, but let me know if you think otherwise.